applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -700,6 +700,10 @@ void slm_at_receive(const uint8_t *buf, size_t len)
 			break;
 		}
 
+#if defined(CONFIG_SLM_UART_ECHO)
+		slm_at_send(buf, len);
```

Comment:
This will print the data as well and should likely call slm_at_send_indicate without indicate and tracing.

Reply:
Now it directly calls at_backend.send